src/compute-types/src/dataflows.rs

```diff
@@ -123,6 +123,12 @@ impl<P, S> DataflowDescription<P, S, mz_repr::Timestamp> {
     ) -> Antichain<mz_repr::Timestamp> {
         let dataflow_expiration_desc = &self.dataflow_expiration_desc;
 
+        tracing::debug!(
```

Comment:
This doesn't include `refresh_schedule`, I think it'd be good to add that, too. What do you think about intercepting the return value and logging that, too?

Reply:
Good point.

Reply:
I don't think this yet logs the return value, right?